plugin/src/App/Components/PatchVisualizer/init.lua

```diff
@@ -183,13 +192,15 @@ function PatchVisualizer:buildTree(patch, instanceMap)
 		-- Gather detail text
 		local changeList, hint = nil, nil
 		if next(change.changedProperties) or change.changedName then
+			local unappliedChanges = findUnappliedPropsForId(unappliedPatch, change.id)
+
 			changeList = {}
 
 			local hintBuffer, i = {}, 0
-			local function addProp(prop: string, current: any?, incoming: any?)
+			local function addProp(prop: string, current: any?, incoming: any?, metadata: any?)
```

Comment:
This is mostly unrelated to this PR but why do we even bother specifying the type of these parameters? It feels like we should either do a better job or not bother, ha ha.

Reply:
Mostly as a reminder to add proper strict typing to everything eventually lol